aws-lambda-layers/node/latest.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
IIRC this file should be a symlink

Reply:
see https://github.com/getsentry/sentry-release-registry/blob/master/bin/sync-links